noaa/gefs/forecast/template.py

```diff
@@ -25,6 +26,13 @@
 
 TEMPLATE_PATH = "noaa/gefs/forecast/templates/latest.zarr"
 
+INIT_TIME_HOURS_TO_FORECAST_LENGTHS = {
+    0: np.timedelta64(840, "h"),
+    6: np.timedelta64(384, "h"),
+    12: np.timedelta64(384, "h"),
+    18: np.timedelta64(384, "h"),
```

Comment:
We're a little inconsistent between xarray's numpy datetime/timedelta and pandas. We can't change xarray, but pandas UX is nicer..  Specific comment for here is if this would work as pd.Timedelta lets do that instead.

Reply:
I changed these lines and ran `update-template`, and there was no diff in the template metadata, so it seems like we can use pandas :)